src/components/calcite-alert/calcite-alert.e2e.ts

```diff
@@ -70,6 +70,38 @@ describe("calcite-alert", () => {
     expect(icon).not.toBeNull();
   });
 
+  it("closes on time based on alert duration", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <div>
```

Comment:
Please use the [`html` helper](https://github.com/Esri/calcite-components/blob/master/src/tests/utils.ts#L119-L137) to have Prettier format embedded HTML:

```js
await page.setContent(html`
    <div>
      ...
`);
```